sops.go

```diff
@@ -207,21 +207,13 @@ func (tree Tree) Decrypt(key []byte, cipher DataKeyCipher, stash map[string][]in
 }
 
 // GenerateDataKey generates a new random data key and encrypts it with all MasterKeys.
-func (tree Tree) GenerateDataKey() ([]byte, error) {
+func (tree Tree) GenerateDataKey() ([]byte, []error) {
```

Comment:
Another option would be to return a single error and fail on the first error. I'm not sure which is best.